### ISSUE TITLE
kvirc fails to find modules and shared directory if installing in another prefix than /usr/ or /usr/local/

### ISSUE THREAD

**Issue by DarthGandalf** [kvirc/KVIrc#178]
**Reported by aleksanteri on 08 Jul 2008 09:51:17 UTC**
KVIrc doesn't seem to find the shared directory or the modules directory if the application has been installed into another directory than /usr/ or /usr/local/. I tried such prefixes as /usr/unstable and /usr/qt4, even /kvi4.

The thing that makes it look like a bug is a part on the module directory's error message:

```
I have been looking for the /modules/libkvisetup.so library but I haven't been able to load it due to the following error: "/modules/libkvisetup.so: cannot open shared object file: No such file or directory"
```

As can be seen, the prefix is completely missing from the error message. 

---

Migrated from: https://svn.kvirc.de/kvirc/ticket/178

**Comment by DarthGandalf** [comment]
**Comment by HelLViS69 on 08 Jul 2008 11:55:27**
Did you try with -DLIB_INSTALL_PREFIX=/path ?

**Comment by DarthGandalf** [comment]
**Comment by anonymous on 08 Jul 2008 21:14:09**
fixed in rev 2043 (trunk 4.0)

---

Resolution: none → fixed
Status: new → closed
